systemlink-notebook-datasource/src/DataSource.ts

```diff
@@ -78,6 +81,7 @@ export class DataSource extends DataSourceApi<NotebookQuery, NotebookDataSourceO
       const parameters = this.replaceParameterVariables(query.parameters, options);
       const execution = await this.executeNotebook(query.path, parameters, query.cacheTimeout);
       if (execution.status === 'SUCCEEDED') {
+        // TODO: this validation doesn't seem to be working
```

Comment:
Did you mean to leave this TODO in? Or maybe it's just an existing issue you noticed while you were here.

Reply:
> it's just an existing issue you noticed while you were here

Yes, this check isn't doing what it should, but I didn't try to fix it in this PR.

Reply:
Do we have a way of tracking issues for this code? Can we associate this TODO with a work item/bug/issue number?